content/en/blog/2022/rust-instrumentation/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: Instrumenting a Rust application with OTel
+linkTitle: Rust Instrumentation
+date: 2022-09-21
+author: '[Yoav Danieli](https://github.com/ydan287) (Aspecto)'
+canonical_url: https://www.aspecto.io/blog/distributed-tracing-with-opentelemetry-rust/
+---
+
+# Getting Started: OpenTelemetry in Rust
```

Comment:
```suggestion
# Getting Started: OpenTelemetry in Rust
```
Seems this heading 1 is not very required since all legacy blogs have put this heading 1 into the front matter as a title.

Reply:
Right, headings in all site page start at level 2, but in this case, the blog title is sufficient, just **drop this heading**.

Reply:
@windsonsea - I'm not sure what your suggested change is, the two lines seem the same.

Reply:
I mean the blogs normally have a style to take the title into front matter at top. Few blogs have a heading 1. 

You can check the blog folder to learn more.

![image](https://user-images.githubusercontent.com/79828097/201446892-863304f7-34e4-48e0-86be-00f0f12da4dd.png)